documentation/docs/98-reference/54-types.md

```diff
@@ -145,26 +164,21 @@ Others are required for SvelteKit to work properly, and should also be left unto
 /// file: .svelte-kit/tsconfig.json
 {
 	"compilerOptions": {
-		// this ensures that types are explicitly
-		// imported with `import type`, which is
-		// necessary as svelte-preprocess cannot
-		// otherwise compile components correctly
-		"importsNotUsedAsValues": "error",
+		// ???
```

Comment:
```suggestion
		// this ensures that types are explicitly
		// imported with `import type`, which is
		// necessary as svelte-preprocess cannot
		// otherwise compile components correctly
```